frontend/packages/dev-console/src/components/import/route/RouteCheckbox.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { CheckboxField } from '../../formik-fields';
+
+const RouteCheckbox: React.FC = () => {
+  return (
```

Comment:
nit: no need for return statement.

Reply:
I'll update this if there is something else to change. I wish that the linter/prettier would just fix this.